CHANGES.md

```diff
@@ -10,6 +10,14 @@ To be released.
 
 ### @fedify/fedify
 
+ - Remove documentLoader property from FederationOptions interface.
+    [[#376],[#393] by Hasang Cho]
+
+    - In fedify/src/federation/federation.ts, removed documentLoader property.
+    - In cli/src/inbox.tsx, switched the inbox command to the loader factory API by replacing the deprecated documentLoader with documentLoaderFactory/contextLoaderFactory and reusing a single instance from getDocumentLoader(), with no functional changes.
+    - In fedify/src/federation/middleware.ts, refactored federationImpl to replace the deprecated documentLoader with documentLoaderFactory, drop the documentLoader-related allowPrivateAddress/userAgent guard, and route loader creation through factories with no intended behavior change.
+    - In fedify/src/federation/middleware.test.ts, migrated tests to the loader factory API (documentLoaderFactory), removed the obsolete allowPrivateAddress+documentLoader assertion, fixed ctx.documentLoader expectations, split mocks (/auth-check vs /object) to avoid URL collisions, and used a rejecting loader in the first context to ensure null on lookup.
```

Comment:
You shouldn't expose internal details of the changes you made in the changelog; it's for users who use Fedify outside the API surface.